pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -103,10 +104,17 @@ func NewController(
 	// 	DeleteFunc: impl.Enqueue,
 	// })
 
-	c.tracker = tracker.New(impl.EnqueueKey, 30*time.Minute)
+	c.tracker = tracker.New(impl.EnqueueKey, 30*time.Second)
```

Comment:
:sweat_smile:

Reply:
oh whoops, just realized i copied the `30*time.Minute` into the PipelineRun controller too...

Reply:
okay silly question but should this lease be > than the reconcile period? if it isn't, it seems like there's a chance the lease runs out before the next reconcile happens? I guess we're talking about seconds tho so maybe it's okay

Reply:
yes, will change that to be a function of the resync period, good catch